vendor/assets/javascripts/behaviors/nested_resources.js

```diff
@@ -135,9 +135,15 @@ Cmsify.NestedResource.prototype.deserializeFields = function(fields) {
 };
 
 Cmsify.NestedResource.prototype.setIsModified = function () {
-  if (tinyMCE.editors.some(function(editor) { return editor.isDirty() }) ||
-      JSON.stringify(this.serializeFields()) != JSON.stringify(this.lastData)) {
+  var tinyMCEDirty = false;
+  tinyMCE.editors.some(function(editor) {
+    if (editor.isDirty()) tinyMCEDirty = true;
+  }.bind(this));
+
+  if (JSON.stringify(this.serializeFields()).localeCompare(JSON.stringify(this.lastData)) != 0) {
```

Comment:
It actually doesn't, it's the other part with parsing the `isDirty()` it was just the first thing I tried and I didn't change it back. I'll change it back.